synapse/lib/layer.py

```diff
@@ -4599,6 +4595,15 @@ async def _editNodeEdgeAdd(self, nid, form, edit, sode, meta):
         else:
             n2formabrv = self.core.setIndxAbrv(INDX_FORM, n2form)
 
+        if (n1cnts := sode['n1verbs'].get(verb)) is None:
```

Comment:
Could we use a default dict of default dicts for this? ( i think it's a hair faster than the conditional logic being in python ). We may have a previous convention of `dictdict()` or soemthing like that ;)

Reply:
I'm not sure there's really a good way to do that since once we msgpack the sode they come back out as normal dicts.